firefox-ios/Client/Frontend/PasswordManagement/DevicePasscodeRequiredViewController.swift

```diff
@@ -78,4 +77,11 @@ class DevicePasscodeRequiredViewController: SettingsViewController {
         viewController.url = SupportUtils.URLForTopic("manage-saved-passwords-firefox-ios")
         navigationController?.pushViewController(viewController, animated: true)
     }
+
+    override func applyTheme() {
+        super.applyTheme()
+
+        let currentTheme = themeManager.currentTheme(for: windowUUID).colors
+        learnMoreButton.setTitleColor(currentTheme.actionPrimary, for: .normal)
```

Comment:
could we use `learnMoreButton.applyTheme` here to be consistent with the other LinkButtons?

Reply:
Sure, I've updated the pull request.  I also updated `PasswordManagerOnboardingViewController` so that the learn more and continue buttons are also calling the `applyTheme` method instead of applying the colors directly to the buttons.

Reply:
thank you!